instance/static/html/instance/index.html

```diff
@@ -50,7 +50,13 @@
                 |
                 <span>Studio: <a href="{{ selected.instance.studio_url }}" target="_new">{{ selected.instance.studio_url }}</a></span>
               </li>
-              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+              <li>
+                <a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> {{ selected.instance.branch_name }}
+                {% if selected.instance.github_pr_url %}
+                | <a href="{{ selected.instance.github_pr_url }}" target="_new">PR#{{ selected.instance.github_pr_number }}</a>
+                {% endif %}
+              </li>
+              {% endif %}
```

Comment:
@smarnach Yup, that seem like good instructions to add to the README - starting from the test environment can get things to start. For having everything working, there is not really a way around going through the configuration file, and setting the right values/creating the right accounts. Configuration seem a recurring pain point, you're not the first one to mention it - but not sure how to do much better than this.